projects/community-payback-and-delius/src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/AppointmentsIntegrationTest.kt

```diff
@@ -194,6 +195,41 @@ class AppointmentsIntegrationTest {
             .andExpect { status().is4xxClientError }
     }
 
+    @Test
+    fun `returns 2xx when limited access check passes but with current restriction flag true`() {
+        val appointmentId = UPWGenerator.LAO_RESTRICTED_UPW_APPOINTMENT.id
+        val response = mockMvc
+            .perform(get("/projects/N01DEFAULT/appointments/$appointmentId?username=LimitedAccess").withToken())
+            .andExpect(status().is2xxSuccessful)
+            .andReturn().response.contentAsJson<AppointmentResponse>()
+        assertThat(response.case.currentRestriction)
+        assertThat(response.case.restrictionMessage)
+    }
+
+    @Test
+    fun `returns 2xx when limited access check fails but with current restriction flag true`() {
+        val appointmentId = UPWGenerator.LAO_RESTRICTED_UPW_APPOINTMENT.id
+        val response = mockMvc
+            .perform(get("/projects/N01DEFAULT/appointments/$appointmentId?username=FullAccess").withToken())
+            .andExpect(status().is2xxSuccessful)
+            .andReturn().response.contentAsJson<AppointmentResponse>()
+        assertThat(response.case.currentRestriction)
+        assertThat(response.case.restrictionMessage.isNullOrEmpty())
```

Comment:
I think these are missing the assertions after `assertThat`?
```suggestion
        assertThat(response.case.currentRestriction).isFalse
        assertThat(response.case.restrictionMessage).isNullOrEmpty()
```